LOG_CHANNEL() is only declared when logging is enabled; with
ENABLE_JOURNALD_LOG=OFF both LOG and RELEASE_LOG are disabled on OpenBSD.

Index: Source/WebDriver/WebDriverService.cpp
--- Source/WebDriver/WebDriverService.cpp.orig
+++ Source/WebDriver/WebDriverService.cpp
@@ -380,6 +380,7 @@ bool WebDriverService::findCommand(HTTPMethod method, 
 void WebDriverService::handleRequest(HTTPRequestHandler::Request&& request, Function<void (HTTPRequestHandler::Response&&)>&& replyHandler)
 {
     Function<void (HTTPRequestHandler::Response&&)> actualReplyHandler = WTF::move(replyHandler);
+#if !RELEASE_LOG_DISABLED
     if (LOG_CHANNEL(WebDriverClassic).state != WTFLogChannelState::Off) {
         RELEASE_LOG_INFO(WebDriverClassic, "HTTP request %s %s (body=%zu bytes)", request.method.utf8().data(), request.path.utf8().data(), request.dataLength);
         actualReplyHandler = [startTime = MonotonicTime::now(), replyHandler = WTF::move(actualReplyHandler)](HTTPRequestHandler::Response&& response) mutable {
@@ -387,6 +388,7 @@ void WebDriverService::handleRequest(HTTPRequestHandle
             replyHandler(WTF::move(response));
         };
     }
+#endif
 
     auto method = toCommandHTTPMethod(request.method);
     if (!method) {
